src/main/java/net/pms/dlna/DLNAMediaDatabase.java

```diff
@@ -457,14 +536,31 @@ private Double toDouble(ResultSet rs, String column) throws SQLException {
 	public synchronized void insertData(String name, long modified, int type, DLNAMediaInfo media) {
 		Connection conn = null;
 		PreparedStatement ps = null;
+
+		boolean existingId = false;
+		if (media != null && media.getDatabaseFileId() > -1) {
+			existingId = true;
+		}
+
 		try {
 			conn = getConnection();
-			ps = conn.prepareStatement(
-				"INSERT INTO FILES(FILENAME, MODIFIED, TYPE, DURATION, BITRATE, WIDTH, HEIGHT, SIZE, CODECV, "+
-				"FRAMERATE, ASPECT, ASPECTRATIOCONTAINER, ASPECTRATIOVIDEOTRACK, REFRAMES, AVCLEVEL, BITSPERPIXEL, "+
-				"THUMB, CONTAINER, MODEL, EXPOSURE, ORIENTATION, ISO, MUXINGMODE, FRAMERATEMODE, STEREOSCOPY, "+
-				"MATRIXCOEFFICIENTS, TITLECONTAINER, TITLEVIDEOTRACK, VIDEOTRACKCOUNT, IMAGECOUNT, BITDEPTH) VALUES "+
-				"(?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?)");
+
+			if (existingId) {
```

Comment:
I think this part of the code was made in order to reuse FILES_STATUS entries, but I can't think of a good reason for it to be honest :) Feel free to favor your changes over mine